<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileLoader</class>
 <widget class="TaurusWidget" name="fileLoader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Files</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="2" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="fineTuneLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QGridLayout" name="fineTuneLayout">
     <item row="0" column="3">
      <widget class="QLabel" name="locationLabel">
       <property name="text">
        <string>RF plant:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="beamCurrentLabel">
       <property name="text">
        <string>Beam current</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QComboBox" name="facadeCombo">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="facadeLabel">
       <property name="text">
        <string>Facade:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="locationCombo">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="beamCurrentValue">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="suffix">
        <string> mA</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="diagGroup">
     <property name="title">
      <string>Diagnostics file name:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLineEdit" name="diagFileName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="diagFileDialog">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="loopsGroup">
     <property name="title">
      <string>Loops file name:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="loopsFileName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="loopsFileDialog">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaurusWidget</class>
   <extends>QWidget</extends>
   <header>taurus.qt.qtgui.container</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
